test/Microsoft.TestPlatform.CrossPlatEngine.UnitTests/DataCollection/InProcDataCollectionExtensionManagerTests.cs

```diff
@@ -53,6 +53,25 @@ public InProcDataCollectionExtensionManagerTests()
         _inProcDataCollectionManager = new TestableInProcDataCollectionExtensionManager(_settingsXml, _mockTestEventsPublisher.Object, _defaultCodebase, _testPluginCache, _mockFileHelper.Object);
     }
 
+    [TestMethod]
+    public void CheckExtensionPathsDuplicates()
+    {
+        var testPluginCache = new TestPluginCache();
+        testPluginCache.UpdateExtensions(new List<string> { @"c:\test1\Collector.dll" }, false);
+        testPluginCache.UpdateExtensions(new List<string> { @"c:\test1\Collector.dll", @"c:\test2\Collector.dll" }, true);
+
+        var inProcDataCollectionExtensionManager = new TestableInProcDataCollectionExtensionManager(_settingsXml, _mockTestEventsPublisher.Object, _defaultCodebase, testPluginCache, _mockFileHelper.Object);
+
+        Assert.AreEqual(3, inProcDataCollectionExtensionManager.CodeBasePaths.Count); // CodeBasePaths contains the two extensions(after removing Deduplicates) and the _defaultCodebase
+
+        Assert.IsTrue(inProcDataCollectionExtensionManager.CodeBasePaths.Contains(_defaultCodebase));
+        Assert.IsTrue(inProcDataCollectionExtensionManager.CodeBasePaths.Contains(Path.GetDirectoryName(@"c:\test1\Collector.dll")));
+        Assert.IsTrue(inProcDataCollectionExtensionManager.CodeBasePaths.Contains(Path.GetDirectoryName(@"c:\test2\Collector.dll")));
+
+
+    }
```

Comment:
Nit: blank lines are not needed.

```suggestion
    }
```